src/main/com/cw/util/FileReaderUtil.java

```diff
@@ -0,0 +1,58 @@
+package main.com.cw.util;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+ * Created by cwijayasundara on 23/01/2017.
+ * This class is a generic class to read any file from the file system which can be re used.
+ */
+
+public class FileReaderUtil {
+
+    // default constructor
+    public FileReaderUtil(){
+
+    }
+
+    public List<String> readFile(String path){
```

Comment:
Seriously? 
Does this problem demands not to use any library for these purposes? 
Even then you could have considers something like https://www.mkyong.com/java8/java-8-stream-read-a-file-line-by-line/